### PR TITLE
Adding a support page

### PR DESCRIPTION
**Pull request by sfshaza2**
Fixes https://github.com/flutter/website/issues/11110

cc @sethladd

### CODE REVIEW

**Comment by sethladd** [comment]
Thank you! I'm still learning how to navigate the repos and the sites (marketing, and docs). This is for the Marketing site, right?

**Comment by sethladd** [comment]
cc @jmagman who also had a vision for a page like this and might have some comments

File: src/content/resources/support.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+they are a great resource.
+
+[Google Developer Experts]: https://developers.google.com/community/experts/directory?specialization=dart%2Cflutter
+
+### I want to get paid support from Google or the Flutter project
+
+As an open source project,
+we're not set up to offer paid support.
+We recommend checking out the
+Flutter experts](https://developers.google.com/community/experts/directory?specialization=dart%2Cflutter)
```
Comment (jmagman) [inline src/content/resources/support.md:50]:
```suggestion
[Flutter experts](https://developers.google.com/community/experts/directory?specialization=dart%2Cflutter)
```

**Comment by jmagman** [comment]
> cc @jmagman who also had a vision for a page like this and might have some comments

Oh perfect, this is what I was hoping for!  A friend plugged into the startup/VC community saw "I need help with Flutter" questions on his forums/group chats, and I couldn't find a single place to tell him to send them.

Thank you for starting this, @sfshaza2 and @sethladd!

**Comment by sethladd** [comment]
If you can think of other cases where "I need help" would apply (e.g., help with Foo or help with Bar), let us know! We'll soon have a place to put it :)

**Comment by sfshaza2** [comment]
No, @sethladd, this page was intended for docs.flutter.dev. If you want it on flutter.dev, you have to speak with Kelvin.

Should I continue with this page?

**Comment by sethladd** [comment]
I'm so sorry! I need to remember that I have to be specific when talking about "flutter.dev". Thanks for your patience here.

IMO this content and this page applies to both docs.flutter.dev and the marketing site flutter.dev. It should be the same content. So , let's proceed!

Meanwhile, I will open a marketing intake request with Kelvin to get this exact same content onto the marketing site flutter.dev. Glad you're pioneering what this page will look at, we'll duplicate it over on the marketing site.

**Comment by sfshaza2** [comment]
I've taken @jmagman's suggestion. It the page otherwise good enough to post? It _would_ be nice to add a graphic of Dash, I think.

**Review by sethladd — COMMENTED** [review]
omg apparenly I forgot to press submit review

**Comment by sfshaza2** [comment]
@parlough, this PR is failing in the tests. When I look at the test results, it seems to be looking for a /resources/support/index file. However, I added this as a /resources/support.md file. And that's how I specified it in the sidenav. What am I missing here?

File: src/content/resources/support.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+[Stack Overflow](https://stackoverflow.com/questions/tagged/flutter),
+or [Discord](https://discord.com/invite/rflutterdev).
+While the community is active in these channels,
+there is no official SLA on responding to new posts
+or chats in these channels.
+
+### I want professional services support with Flutter
+
+Check out the
+[Flutter consultants]({{site.main-repo}}/consultants).
```
Comment (sethladd) [inline src/content/resources/support.md:34]: nit: the Flutter consultants _directory_? or _list_?

seems like we might be missing a word after "consultants" ?

**Review by sethladd — APPROVED** [review]
LGTM with comment

File: src/content/resources/support.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+There are many different ways to get support
+when developing with Flutter.
+As Flutter is an open source project,
+your options range from filing issues when you
+run into bugs in the software,
+to connecting with your local community,
+to browsing and identifying a list of commercial agencies
+and consultants who offer support services.
+
+### I want to report an issue with Flutter
```
Comment (parlough) [inline src/content/resources/support.md:17]: The checks and build are failing as these headers aren't under a h2. Perhaps they all should be h2 instead?

**Review by parlough — APPROVED** [review]
Thanks!

Looks good to me after the headers are fixed :D

File: src/content/resources/support.md (hunk 3 above)
Comment (sfshaza2) [reply src/content/resources/support.md:17]: Oh! I didn't know that we were enforcing that.

File: src/content/resources/support.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+they are a great resource.
+
+[Google Developer Experts]: https://developers.google.com/community/experts/directory?specialization=dart%2Cflutter
+
+## I want to get paid support from Google or the Flutter project
+
+As an open source project,
+we're not set up to offer paid support.
+We recommend checking out the
+[consultancy program]({{site.main-repo}}/consultants).
```
Comment (parlough) [inline src/content/resources/support.md:54]:
```suggestion
[consultancy program]({{site.main-url}}/consultants).
```

File: src/content/resources/support.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+[Stack Overflow](https://stackoverflow.com/questions/tagged/flutter),
+or [Discord](https://discord.com/invite/rflutterdev).
+While the community is active in these channels,
+there is no official SLA on responding to new posts
+or chats in these channels.
+
+## I want professional services support with Flutter
+
+Check out the
+[Flutter consultants]({{site.main-repo}}/consultants)
```
Comment (parlough) [inline src/content/resources/support.md:34]:
```suggestion
[Flutter consultants]({{site.main-url}}/consultants)
```

**Pull request merged**